lib/supabase/fetchPosts.ts

```diff
@@ -24,6 +24,9 @@ const fetchPosts = async (
   if (!query) {
     return { posts: [] };
   }
+
+  query.filter('author', 'cs', '{"power_badge": true}');
```

Comment:
can you update so this filter doesn't apply on the `following` tab?
- only for `Trending` and `Recent`